modules/sno-du-cluster-specific-addressing.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-cluster-specific-addressing_{context}"]
+= Configuring cluster-specific addressing
+
+After networking has been configured, configure the specific addresses for the single node
+distributed unit (DU).
+
+.Procedure
+
+* The following example shows the configuration for IPv4 and IPv6 addresses. Configure IPv4 and IPv6 on the network interface to support a dual stack deployment. Customize these settings for the specific environment of the single node DU:
++
+[source,yaml]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: NMStateConfig
+metadata:
+  labels:
+    nmstate-label: sno-du-1
+  name: sno-du-1
+  namespace: sno-du-1
+spec:
+  config:
+    dns-resolver:
+      config:
+        server:
+          - 10.19.42.41 <1>
+    interfaces:
+      - ipv4:
+          address: <2>
+            - 10.16.231.3/24
+            - 10.16.231.28/24
+            - 10.16.231.31/24
+        dhcp: false
+          enabled: true
+        ipv6:
+            address: <3>
+              - 2620:52:0:10e7:e42:a1ff:fe8a:601/64
+              - 2620:52:0:10e7:e42:a1ff:fe8a:602/64
+              - 2620:52:0:10e7:e42:a1ff:fe8a:603/64
+        dhcp: false
+          enabled: true
+        macAddress: "00:00:00:01:20:30"
+          state: up
+          name: eno1
+          type: ethernet
+        routes:    <4>
+          config:
+            - destination: 0.0.0.0/0
+                next-hop-address: 10.16.231.254
+                next-hop-interface: eno1
+                table-id: 254
+    interfaces: <5>
+      - name: eno1
+          macAddress: "00:00:00:01:20:30"
+----
+<1> Configure the DNS entries.
+<2> Configure `eno1` IPv4 connectivity. In this case, it is assumed there is a dual stack.
+<3> Configure `eno1` IPv6 connectivity. In this case, it is assumed there is a dual stack.
+<4> The default route configuration.
+<5> The interface name and MAC address.
```

Comment:
```suggestion
apiVersion: agent-install.openshift.io/v1beta1
kind: NMStateConfig
metadata:
  labels:
    nmstate-label: sno-du-1
  name: sno-du-1
  namespace: sno-du-1
spec:
  config:
    dns-resolver:
      config:
        server:
          - 10.19.42.41 <1>
    interfaces:
      - ipv4:
          address: <2>
            - 10.16.231.28/24
          dhcp: false
          enabled: true
        ipv6:
          enabled: false
        macAddress: "00:00:00:01:20:30"
        state: up
        name: eno1
        type: ethernet
    routes:    <3>
      config:
        - destination: 0.0.0.0/0
            next-hop-address: 10.16.231.254
            next-hop-interface: eno1
            table-id: 254
  interfaces: <4>
    - name: eno1
      macAddress: "00:00:00:01:20:30"
----
<1> Configure the DNS entries.
<2> Configure `eno1` IPv4 connectivity.
<3> The default route configuration.
<4> The interface name and MAC address.
```

Reply:
Ignore previous. Too many indentation issues to recover from, here is a clean generated version:
```suggestion
apiVersion: agent-install.openshift.io/v1beta1
kind: NMStateConfig
metadata:
    annotations:
        argocd.argoproj.io/sync-wave: "1"
    labels:
        nmstate-label: cnfocto1
    name: cnfocto1.ptp.lab.eng.bos.redhat.com
    namespace: cnfocto1
spec:
    config:
        dns-resolver:
            config:
                search:
                    - ptp.lab.eng.bos.redhat.com
                server:
                    - 10.19.42.41 <1>
        interfaces:
            - ipv4:
                address:
                    - ip: 10.16.231.28 <2>
                      prefix-length: 24
                enabled: true
              ipv6:
                enabled: false
              macAddress: 00:00:00:01:20:30
              name: eno1
              state: up
              type: ethernet
        routes:
            config:
                - destination: 0.0.0.0/0 <3>
                  next-hop-address: 10.16.231.254
                  next-hop-interface: eno1
                  table-id: 254
    interfaces: <4>
        - name: eno1
          macAddress: 00:00:00:01:20:30

----
<1> Configure the DNS entries.
<2> Configure `eno1` IPv4 connectivity.
<3> The default route configuration.
<4> The interface name and MAC address.
```

Reply:
Made this update.